corehq/apps/reports/templates/reports/filters/bootstrap2/drilldown_options.html

```diff
@@ -35,6 +35,7 @@
 {% endblock %}
 {% block filter_js %}
 <script type="text/javascript" src="{% static 'reports/ko/report_filter.drilldown_options.js' %}"></script>
+{% if not is_empty %}
```

Comment:
not crazy about the variable name here (is *what* empty?), but I see it's already defined on the page, so I won't make you fix it.